roles/osp_observability/defaults/main.yml

```diff
@@ -5,3 +5,9 @@ workdir: /var/lib/osp-observability
 
 openstack_cloud_name: overcloud
 prometheus_scrape_interval: "1s"
+
+#sg-vars
+ceilometer_port: '6420'
```

Comment:
The naming of this variable tripped me up. 

`prometheus_port` is the port prometheus is listening on, so I would expect`ceilometer_port` is the port ceilometer is listening on. Instead I think this is the port that _sg_core is listening on_ for client connection from ceilometer.

Suggest `sg_ceilometer_port`

Reply:
Agree.